src/Umbraco.Core/Strings/Utf8ToAsciiConverter.cs

```diff
@@ -3560,29 +3560,55 @@ private static void ToAscii(ReadOnlySpan<char> input, int ipos, Span<char> outpu
                     output[opos++] = 'a';
                     break;
 
+
                 // BEGIN EXTRA
-                /*
-            case '£':
-                output[opos++] = 'G';
-                output[opos++] = 'B';
-                output[opos++] = 'P';
-                break;
-
-            case '€':
-                output[opos++] = 'E';
-                output[opos++] = 'U';
-                output[opos++] = 'R';
-                break;
-
-            case '©':
-                output[opos++] = '(';
-                output[opos++] = 'C';
-                output[opos++] = ')';
-                break;
-                */
+                //Currencies: https://cloford.com/resources/charcodes/utf-8_currency.htm
+                case '£':
+                    output[opos++] = 'G';
+                    output[opos++] = 'B';
+                    output[opos++] = 'P';
+                    break;
+
+                case '€':
+                    output[opos++] = 'E';
+                    output[opos++] = 'U';
+                    output[opos++] = 'R';
+                    break;
+                case '$':
+                    output[opos++] = 'U';
+                    output[opos++] = 'S';
+                    output[opos++] = 'D';
+                    break;
```

Comment:
Maybe consider cent sign as well?
https://www.alt-codes.net/cent_alt_code.php

Reply:
@bjarnef thanks for the suggestion, I was busy adding a lot of the other characters and doing a proper job just adding all the utf-8 currencies. Considering this probably won't merge as per the other comments. Considering this gonna hold off on that for the time being.